_query-dsl/geo-and-xy/geopolygon.md

```diff
@@ -0,0 +1,177 @@
+
+![Search for points within the specified quadrilateral]({{site.url}}{{site.baseurl}}/images/geopolygon-query.png)
+
+The response contains the matching document:
+
+```json
+{
+  "took": 6,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 1,
+      "relation": "eq"
+    },
+    "max_score": 1,
+    "hits": [
+      {
+        "_index": "testindex1",
+        "_id": "1",
+        "_score": 1,
+        "_source": {
+          "point": {
+            "lat": 73.71,
+            "lon": 41.32
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+In the preceding search request, you specified the polygon vertices in clockwise order:
+
+```json
+"geo_polygon": {
+    "point": {
+    "points": [
+        { "lat": 74.5627, "lon": 41.8645 },
+        { "lat": 73.7562, "lon": 42.6526 },
+        { "lat": 73.3245, "lon": 41.6189 },
+        { "lat": 74.0060, "lon": 40.7128 }
+    ]
+    }
+}
+```
+
+Alternatively, you can specify the vertices in counterclockwise order:
+
+```json
+"geo_polygon": {
+    "point": {
+    "points": [
+        { "lat": 74.5627, "lon": 41.8645 },
+        { "lat": 74.0060, "lon": 40.7128 },
+        { "lat": 73.3245, "lon": 41.6189 },
+        { "lat": 73.7562, "lon": 42.6526 }
+    ]
+    }
+}
+```
+
+The resulting query response contains the same matching document.
+
+However, if you specify the vertices in the following order:
+
+```json
+"geo_polygon": {
+    "point": {
+    "points": [
+        { "lat": 74.5627, "lon": 41.8645 },
+        { "lat": 74.0060, "lon": 40.7128 },
+        { "lat": 73.7562, "lon": 42.6526 },
+        { "lat": 73.3245, "lon": 41.6189 }
+    ]
+    }
+}
+```
+
+The response returns no results.
+
+## Request fields
+
+Geopolygon queries accept the following fields.
+
+Field | Data type | Description
+:--- | :--- | :--- 
+`_name` | String | The name of the filter. Optional.
+`validation_method` | String | The validation method. Valid values are `IGNORE_MALFORMED` (accept geopoints with invalid coordinates), `COERCE` (try to coerce coordinates to valid values), and `STRICT` (return an error when coordinates are invalid). Optional. Default is `STRICT`.
+`ignore_unmapped` | Boolean | Specifies whether to ignore an unmapped field. If set to `true`, then the query does not return any documents that contain an unmapped field. If set to `false`, then an exception is thrown when the field is unmapped. Optional. Default is `false`.
+
+## Accepted formats
+
+You can specify the geopoint coordinates when indexing a document and searching for documents in any [format]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-point#formats) accepted by the geopoint field type.  
```

Comment:
Should the second instance of `geopoint` be in code font?

Reply:
No because then it would be `geo_point` 😄